examples/tpm2-ekcert/main.go

```diff
@@ -0,0 +1,59 @@
+// Binary tpm2-ekcert reads an x509 certificate from a specific NVRAM index.
+package main
+
+import (
+	"crypto/x509"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+)
+
+var (
+	tpmPath   = flag.String("tpm-path", "/dev/tpm0", "Path to the TPM device (character device or a Unix socket)")
+	certIndex = flag.Uint("cert-index", 0, "NVRAM index of the certificate file")
+	outPath   = flag.String("output", "", "File path to for output. Leave blank to write to stdout.")
+)
+
+func main() {
+	flag.Parse()
+
+	if *certIndex == 0 {
+		fmt.Println("--cert-index must be set")
```

Comment:
Consider printing errors to stderr:

fmt.Fprintln(os.stderr, "--cert-index must be set")